test/integration/good/code-gen/dune

```diff
@@ -22,7 +22,7 @@
 
 (rule
  (targets transformed_mir.output)
- (deps (package stanc) (:stanfiles mother.stan))
+ (deps (package stanc) (:stanfiles mother.stan unenforce_initialize.stan))
```

Comment:
Nitpicking alert.

Could we just add those example to the mother model?

Reply:
Ummm yeah we can I'm fine with that

Reply:
Cool, move to mother so we only have one model and the we are good to go.

Reply:
Actually I think we have a lot of examples where this is used in the tests already so I'm just going to remove this example since it already is used by mother.stan

Reply:
Yeah, agreed.